src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ReferralServiceTest.kt

```diff
@@ -1,57 +1,71 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.service
-
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.DraftReferral
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository.ReferralRepository
 import java.time.LocalDate
 import java.time.OffsetDateTime
-import java.util.UUID
-
-class ReferralServiceTest {
-  private fun newOriginal(): Referral {
-    return Referral(
-      id = UUID.fromString("3529d30e-3480-4686-b14b-485beb3431ed"),
-      created = OffsetDateTime.parse("2021-05-05T13:00:01+01:00"),
-      // 👇 business fields
-      completionDeadline = LocalDate.parse("2021-06-26")
-    )
-  }
+
+@DataJpaTest
+class ReferralServiceTest @Autowired constructor(
+  val entityManager: TestEntityManager,
+  val referralRepository: ReferralRepository
+) {
+
+  private val referralService = ReferralService(referralRepository)
 
   @Test
   fun `update cannot overwrite identifier fields`() {
-    val update = DraftReferral(
-      id = UUID.fromString("ce364949-7301-497b-894d-130f34a98bff"),
```

Comment:
we have deleted the id field from the incoming draft referral here.

Reply:
Well spotted, was unsure whether to just set the id passed in, into the DraftReferral, but then decided to just pass separately.  Added id back into test